.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,222 @@
+["Insert a suggestion"](https://help.github.com/en/articles/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request)
+    - For the **"reviewee"**, please accept those changes. GitHub will insert the
+    suggestion directly as a commit, thus giving attribution to the reviewer. Note:
+    This does not work for multi-line suggestions edits (though you can add more
+    lines in the suggestion if desired).
+- For individual chapter reviews, look for consistency, content, clarity of
+material with exercises. Don't worry too much about grammar appearance, structure,
+etc. This will be done at the end stage.
+
+## Specific Content Guidelines
+
+1.  bookdown's behavior is controlled by the metadata in `_bookdown.yml` and in the YAML header of `index.Rmd`.
+    Please do not add YAML headers to other `.Rmd` files, since it is ignored when building the entire book.
+
+1.  `_bookdown.yml` contains an ordered list of R Markdown chapters and appendices.
+    This list is repeated in `Makefile`.
+
+1.  The first element of each chapter must be a level-1 heading.
+    Anything before this heading is silently ignored.
+
+1.  The ID for each chapter and section heading is set by putting `{#some-label}` on the heading line.
+    Please use the stem of the file's name in labels, i.e., start all labels for chapter `stuff.Rmd` with the word `stuff`.
+
+1.  Each chapter starts with a list of questions and a list of learning objectives and ends with a list of key points.
+    These lists are kept in Markdown files in the `questions`, `objectives`, and `keypoints` folders respectively
+    so that they can be included in both the chapters and the `objectives.Rmd` and `keypoints.Rmd` appendices.
+
+1.  The last line in each chapter includes the file `links.md`, which gives symbolic names to all the external links used in the book.
+    Please always use `[text][label]` to refer to links and put the URL itself in this file to ensure consistency between chapters.
+
+1.  Please use the free desktop version of [draw.io](https://www.draw.io/) to draw diagrams.
+    Save the source as `figures/stem/stem.drawio` and export drawings as all of PDF, PNG, and SVG.
+
+1.  Since LaTeX doesn't understand SVG images, hand-drawn diagrams are included as follows:
+    -   Create an R code block with the header `{r stem-label, echo=FALSE, fig.cap="Some Caption"}`, which means:
+        -   Give this figure the ID `fig:stem-label`.
+            (Bookdown automatically puts `fig:` in front of figure labels,
+            though it doesn't prefix section labels with anything.)
+        -   Don't show the R code used to load the image, just its output (i.e., the image).
+        -   Give the figure the specified caption.
+    -   Use `insert_graphic("figures/stem/filename.ext")` to include your image. If you are using a `.pdf` image, make sure that you have the same file but as a `.svg` as well. 
+
+1.  Use `@Name1234` to refer to bibliography entries.
```

Comment:
Don't we want `[@Name1234]` so that the citations are surrounded by parentheses? And leave out `[]` only when the citation is truly part of the sentence (the "in-text" [example here](https://rmarkdown.rstudio.com/authoring_bibliographies_and_citations.html#citation_syntax)). Mention `;` for multiple references or just include the link I just posted.

Reply:
I'm not always sure how the [ works, since it depends on the style. I included mention of the [, but don't expand on it.